app/windows/winget/winget_create.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Usage:
+ *
+ * 1. Ensure that Node.js and npm are installed on your system.
+ * 2. In the project's root directory, initialize a package.json file with the command `npm init -y`.
+ * 3. Install the node-fetch module with the command `npm install node-fetch`.
+ * 4. Run this script using the command `node winget_create.js <projectVersion> <sha256>`.
+ *
+ * Parameters:
```

Comment:
We need a destination directory parameter, so we write the new files into that.
I imagine we'll be using it like:
node winget_create.js 0.19.1 /my/path/to/the/wingetrepo/h/headlamp/0.19.1

The given directory should be created if it doesn't exist.